src/clusterfuzz/_internal/system/environment.py

```diff
@@ -1051,7 +1056,7 @@ def bot_noop(func):
 
   @functools.wraps(func)
   def wrapper(*args, **kwargs):
-    is_bot = not is_running_on_app_engine()
+    is_bot = not (is_running_on_app_engine() or is_running_on_k8s())
```

Comment:
I will pass 'REDIS_HOST' value to k8s script for testing purpose. After testing, the environment variable will be set during the deploying stage, same with app engine. https://github.com/google/clusterfuzz/blob/52db8878254a2bff7cfe5fce68815ab00bb64824/src/local/butler/deploy.py#L83

Reply:
Instead of re-using `bot_noop`, we should create a new one:

```python
def if_redis_available(func):
  ...


@if_redis_available
def get(...)
```

`bot_noop` is meant for a different use case (i.e. are we a bot or not). better to separate out the intent of this new chekc.